tasks/13_largest-of-four/largest-of-four.js

```diff
@@ -0,0 +1,3 @@
+export function largestOfFour(arr) {
+  return arr.map((arr) => arr.reduce((max, curr) => curr>max? curr: max), 0);
```

Comment:
Adding comments or splitting this single line solution in the future would be nice in order to increase readability